tox.ini

```diff
@@ -29,6 +29,7 @@ commands = flake8 {posargs} bandit
            bandit-baseline -r bandit -ll -ii
 
 [testenv:pep8]
+ignore_errors = true
```

Comment:
This is because pylint throws tracebacks if we don't upgrade the version, and spews a bunch of new issues if we do. So for now, let's leave it static and upgrade it in a separate PR